src/AppBundle/Membership/MembershipRequestHandler.php

```diff
@@ -57,7 +57,7 @@ public function update(Adherent $adherent, MembershipRequest $membershipRequest)
     {
         $adherent->updateMembership($membershipRequest, $this->addressFactory->createFromAddress($membershipRequest->getAddress()));
 
-        $this->manager->flush();
```

Comment:
What if the address didn't change? We're still regeocoding it and flushing no changes. Fortunately the geocoder in production uses a cache for geocoded addresses.